db/patterns/nexon.eno

```diff
@@ -0,0 +1,16 @@
+name: Nexon Global
+category: site_analytics
+website_url: https://www.nexon.com/main/en
+organization: nexon
+
+--- domains
+ssl.nexon.com
+public.api.nexon.com
+--- domains
+
+--- filters
+||ssl.nexon.com/s1/da/a2s.js
+||public.api.nexon.com/nxlog-collect/apne2/client.all
+--- filters
+
+ghostery_id:
```

Comment:
ghostery_id is optional and when empty can be skipped. It is mostly used for legacy purposes in the Ghostery Extension. For all new patterns the pattern file name is being used as a implicit ghostery_id.

Reply:
Removed `ghostery_id` from both pattern and organization.